crates/rspack_loader_runner/src/runner.rs

```diff
@@ -219,12 +219,9 @@ async fn create_loader_context<'c, C: 'c>(
   plugins: &'c [Box<dyn LoaderRunnerPlugin>],
   context: C,
 ) -> Result<LoaderContext<'c, C>> {
-  let mut file_dependencies: HashSet<PathBuf> = Default::default();
```

Comment:
this code is useless, we add `resource_data.resource_path` as file dependencies when resolve success